packages/sample/package.json

```diff
@@ -6,8 +6,17 @@
     "private": true,
     "type": "module",
     "dependencies": {
+        "@rev-robotics/control-hub": "*",
         "@rev-robotics/expansion-hub": "*",
```

Comment:
The sample should depend on specific versions of these libraries. Remember, we want people to be able to use this sample via `npx`.